website/bin/exr2aces.rst

```diff
@@ -15,26 +15,46 @@ Description
 Read an OpenEXR file from infile and save the contents
 in ACES image file outfile.
 
-The ACES image file format is a subset of the OpenEXR file
-format.  ACES image files are restricted as follows:
+The ACES Image Container File Format specified in `SMPTE ST 2065-4`_ 
+is a subset of the OpenEXR file format defined by the implementation 
+of the time (roughly OpenEXR 1.6). 
 
-- Images are stored as scanlines; tiles are not allowed.
+This imposes considerable restrictions on what image data and metadata 
+can be transported from a modern OpenEXR file into an ACES Image 
+Container File, including (but not limited to) the following:
 
-- Images contain three color channels, either R, G, B (red, green,
-  blue) or Y, RY, BY (luminance, sub-sampled chroma)
+- The version field must be either be 2 or 1026; a value of 2 implies 
+attribute names cannot be longer than 31 characters in length, 
```

Comment:
```suggestion
   attribute names cannot be longer than 31 characters in length, 
   as was the case prior to OpenEXR 1.7.
```

Reply:
Need to indent the lines below the '-' so they're justified underneath the first character after the space after the '-'.

Reply:
sorry about that, fixed!